.github/scripts/npm_metrics.py

```diff
@@ -0,0 +1,38 @@
+"""
+Submit metrics returns "Payload accepted" response
+"""
+
+from datetime import datetime
+from datetime import timedelta
+from datadog_api_client.v1 import ApiClient, Configuration
+from datadog_api_client.v1.api.metrics_api import MetricsApi
+from datadog_api_client.v1.model.metrics_payload import MetricsPayload
+from datadog_api_client.v1.model.point import Point
+from datadog_api_client.v1.model.series import Series
+import requests
+
+today = datetime.today()
+yesterday = today - timedelta(days = 1)
+yesterday_url_format= yesterday.strftime('%Y-%m-%d')
+npm_url="https://api.npmjs.org/downloads/point/"+yesterday_url_format+"/twilio-cli"
+
+response = requests.get(npm_url)
+download_count=response.json()
+download_count_float=float(download_count['downloads'])
+
+body = MetricsPayload(
+    series=[
+        Series(
+            metric="npm.download.count",
```

Comment:
Is it a common Datadog bucket for metrics or are these metric names in global namespace for Twilio org? 
If it's global- I'd suggest naming the metrics with a twilio-cli prefix.

Reply:
Should be a separate one as we have a unique API key. The others won't be able to see this